src/cfg/cfg-traversal.h

```diff
@@ -45,7 +45,13 @@ struct CFGWalker : public ControlFlowWalker<SubType, VisitorType> {
     std::vector<BasicBlock*> out, in;
   };
 
-  BasicBlock* entry; // the entry block
+  // The entry block at the function's start. This always exists.
+  BasicBlock* entry;
+
+  // The exit block at the end where control flow leaves the function. This may
+  // not exist if nothing flows out directly (for example, if the function ends
+  // in a return or an unreachable).
+  BasicBlock* exit;
```

Comment:
- If a function ends in a return, is that not considered "flowing out directly"?
- What if there are multiple exit points?

Reply:
I improved the comment now, PTAL. I agree it was not clear enough before. It should address your questions here I hope.